gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-ktlint = "1.5.0"
+ktlint = "1.3.0"
```

Comment:
Do we really have to downgrade our ktlint version to get this fix? If so, could we open an issue to Ktlint to fix this problem, so we are unblocked for future upgrades?